content/en/docs/setup/best-practices/certificates.md

```diff
@@ -134,7 +134,7 @@ You must manually configure these administrator account and service accounts:
 | admin.conf              | default-admin              | kubernetes-admin               | system:masters |
 | kubelet.conf            | default-auth               | system:node:`<nodeName>` (see note) | system:nodes   |
 | controller-manager.conf | default-controller-manager | system:kube-controller-manager |                |
-| scheduler.conf          | default-manager            | system:kube-scheduler          |                |
+| scheduler.conf          | default-scheduler          | system:kube-scheduler          |                |
```

Comment:
the change seems fine.
the important part is passing the same "name" to the kubectl commands bellow.
one can also use the "Default CN" values for that.

i agree that `default-manager` is a bit confusing.

Reply:
Happy to /lgtm @neolit123 ?

Reply:
/lgtm